addons/mail/static/src/views/web/fields/html_composer_message_field/content_expandable_plugin.js

```diff
@@ -70,6 +70,7 @@ export class ContentExpandablePlugin extends Plugin {
         for (const subEl of ele.querySelectorAll(":scope > .o_mail_reply_content")) {
             subEl.classList.toggle("d-none");
         }
+        closestElement(ev.target, "div").remove();
```

Comment:
could you use a more precise selector:
`div.o-mail-Message-viewMore-container`
instead?